content/guides/models/track/runs/_index.md

```diff
@@ -315,6 +339,32 @@ https://wandb.ai/<team-name>/<project-name>/runs/<run-id>
 
 Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and run ID.
 
+### Customize how runs are displayed
+You can customize how runs are displayed in your project from the **Workspace** or **Runs** tab, which share display settings.
+
```

Comment:
Found this part a little confusing: "...runs are displayed in your project from the **Workspace** or **Runs** tab, which share display settings". Maybe break up into two sentences?

Reply:
PTAL about my update